code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -63,6 +125,23 @@
 	text_dehack = "[name]'s software has been reset!"
 	text_dehack_fail = "[name] does not seem to respond to your repair code!"
 
+/mob/living/simple_animal/bot/cleanbot/Crossed(atom/movable/AM)
+	if(has_gravity() && ismob(AM) && weapon)
```

Comment:
Shouldn't there be a check for if the victim is on the ground here? Otherwise someone can just continuously drag you onto the cleanbot and off again for a quick kill